onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -377,6 +377,27 @@ ONNX_MS_OPERATOR_SET_SCHEMA(FastGelu, 1,
                                   return true;
                                 }));
 
+ONNX_MS_OPERATOR_SET_SCHEMA(RelPosAttnBiasGen, 1,
+                            OpSchema()
+                                .SetDoc("Compute binned relative position bias")
+                                .Attr("max_distance", "Max distance", AttributeProto::INT)
+                                .Attr("is_bidirectional", "Default value is 0.", AttributeProto::INT, static_cast<int64_t>(0))
+                                .Input(0, "bias_table", "2D input tensor with shape (num_buckets, num_heads), COL-major(See UT for example)", "T")
+                                .Input(1, "sequence_length", "Real sequence length from last dimension of hidden states", "U")
```

Comment:
"Real" seq length usually means after removing padding. But the last dimension of hidden states usually the length with padding. Maybe update the comment to clarify.